### ISSUE TITLE
semicolon in comment breaks query

### ISSUE THREAD

**Issue by jon-rtr** [fishtown-analytics/dbt#711]
_tl;dr Semicolons in sql comments in a model break dbt._

I am executing `dbt run --models orders`, which is a model I directly ported from our legacy ETL infrastructure onto dbt.

The following error is appearing:
```
Database Error in model orders (models/transformed/orders.sql)
  001003 (42000): SQL compilation error:
  syntax error line 48 at position 40 unexpected '<EOF>'.
  compiled SQL at target/run/analytics/transformed/orders.sql
```

The following files are generated and look correct:
- target/compiled/analytics/transformed/orders.sql
- target/run/analytics/transformed/orders.sql

I logged into Snowflake to see what query was actually executed and discovered that the only a portion of the original query was sent to the database, the text before a semicolon appearing in a comment.

Here is a minimal test case that exercises the failure:
```
select -- this is; only a test
    1 as qux
```

My workaround is to remove any semicolons from the sql comments.

**Comment by drewbanin** [comment]
@jon-rtr thanks for making this! This is definitely an issue on Snowflake -- we've been working around it by just removing semicolons from comments like you but... that's not a very good solution.

Thanks for creating this issue!

**Comment by drewbanin** [comment]
can we use `execute_stream` ([here](https://docs.snowflake.net/manuals/user-guide/python-connector-example.html)) to run multiple queries in the same connection as we do with other dbs?
